example-use-cases/archive-wide-queries/fetch_initial_release_date.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import csv
+from dateutil import parser
+from rcsbapi.data import DataQuery as Query
```

Comment:
I would add a header at the top of this file describing what it does and a reminder to install the packages used. E.g.:

```python
"""
Python script for fetching the initial release date for all PDB entries (via RCSB.org).
The results are saved into a CSV file, `rcsb_release_dates.csv`.

This script requires the following packages, which can be installed with:
pip install python-dateutil
pip install requests
pip install rcsb-api
"""
```